transposer/plugins/org.polarsys.kitalpha.transposer.ui/src/org/polarsys/kitalpha/transposer/ui/internal/tabs/TransposerMainTab.java

```diff
@@ -60,326 +64,458 @@
  */
 public class TransposerMainTab extends AbstractLaunchConfigurationTab {
 
-  private static final String TAB_NAME = "Main"; //$NON-NLS-1$
```

Comment:
There is a line ending encoding problem on this page, can you please fix it ?

Reply:
could you explain the problem at this line in details!

Reply:
My mistake, my comment is not understandable. I think the whole file is unchanged, only the line endings have changed, so my guess would be that it should not be commited.

Reply:
It seems that there is also a lot of formatting in this file change. So maybe it is a good thing to modify it.

Reply:
No. The main changes I commited was here :). See for example: reateGenerationPathWidget(...)

Reply:
Ok I see it now. Thanks.